data/migrations/deb/1_3_428_to_1_3_429.py

```diff
@@ -77,17 +138,23 @@ def migrate_txn_log(db_dir, db_name):
     # put values from old ledger to the new one
     try:
         for key, val in src_storage.iterator():
+            key = key.decode()
             val = ledger_txn_serializer.deserialize(val)
             new_val = transform_to_new_format(txn=val, seq_no=int(key))
+            digest = put_into_seq_no_db(new_val)
+            # add digest into txn
+            new_val[TXN_PAYLOAD][TXN_PAYLOAD_METADATA][TXN_PAYLOAD_METADATA_DIGEST] = digest
```

Comment:
Should we do it for genesis txns as well? If we do, then we will have to migrate sovrin genesis txns...